fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/sfcc.rs

```diff
@@ -385,46 +358,13 @@ impl SlatFlapControlComputer {
     }
 
     fn slat_actual_position_word(&self) -> Arinc429Word<f64> {
-        Arinc429Word::new(
-            self.slats_feedback_angle.get::<degree>(),
-            SignStatus::NormalOperation,
-        )
+        let slats_fppu = self.slats_channel.get_feedback_angle();
+        Arinc429Word::new(slats_fppu.get::<degree>(), SignStatus::NormalOperation)
     }
 
     fn flap_actual_position_word(&self) -> Arinc429Word<f64> {
-        Arinc429Word::new(
-            self.flaps_feedback_angle.get::<degree>(),
-            SignStatus::NormalOperation,
-        )
-    }
-}
-
-trait SlatFlapLane {
-    fn signal_demanded_angle(&self, surface_type: &str) -> Option<Angle>;
-}
-
-impl SlatFlapLane for SlatFlapControlComputer {
-    fn signal_demanded_angle(&self, surface_type: &str) -> Option<Angle> {
-        match surface_type {
-            "FLAPS"
-                if Self::surface_movement_required(
-                    self.flaps_demanded_angle,
-                    self.flaps_feedback_angle,
-                ) =>
-            {
-                Some(self.flaps_demanded_angle)
-            }
-            "SLATS"
-                if Self::surface_movement_required(
-                    self.slats_demanded_angle,
-                    self.slats_feedback_angle,
-                ) =>
-            {
-                Some(self.slats_demanded_angle)
-            }
-            "FLAPS" | "SLATS" => None,
-            _ => panic!("Not a valid slat/flap surface"),
-        }
+        let flaps_fppu = self.flaps_channel.get_feedback_angle();
```

Comment:
```suggestion
        let flaps_angle = self.flaps_channel.get_feedback_angle();
```

Reply:
Either or should work. I think the `fppu` part is not that important (also not on line 297 actually).